### ISSUE TITLE
Unable to edit/save a  .kdbx file from a shared folder

### ISSUE THREAD

**Issue by ruihildt** [jhass/nextcloud-keeweb#44]
I have a .kdbx ile I use for organization passwords, between admin1 and admin2.

When I try to edit the file with admin2, I can add, but autosave doesn't work and if I try to save manually the message in the UI is  `[object Event]`.

Is there something I'm missing or is it a bug?

**Comment by bessangel** [comment]
For sharing password try use https://github.com/nextcloud/passman#readme
it is better for this purpose IMHO.

**Comment by ruihildt** [comment]
But do I need to install passman separately from my nextcloud instance?

**Comment by bessangel** [comment]
No, you dont need. I use it from nextcloud web gui, as another nextcloud application, it is buildin nextcloud application without extrenal dependensies. Only if you want use passman password database in some desktop gui application in same time, this will be problem may be, because I didn`t find where passman keep the database.
But next features is more useable in passman: sharing password with more people at once, sharing password by url. We use passman for keep system and company password (system administrations purposes), and keeweb for personal password security.

**Comment by mhow2** [comment]
@bessangel : 

> For sharing password try use https://github.com/nextcloud/passman#readme

How do you share passwords with passman as it is designed is per-user ? I didn't find a way to share a passman vault with others nextcloud users...

**Comment by DecaTec** [comment]
Opening a kdbx from a shared link would be a great enhancement.
This way, you can download the kdbx and use your desktop application or open the file directly in Nextcloud.

**Comment by maurerle** [comment]
So there are two things:

1. opening a kdbx from a shared link (ex.: `https://nextcloud.instance/s/xTGa4GH2ziABsA4`) does not open keeweb. This is working for pdfs and music files so this should be doable somehow.

2. sharing a kdbx file with another user opens keeweb but saving shows an error in keeweb. With the sync error message in keeweb UI `[object Event]`.

So this should be two seperate issues.

**Comment by SimJoSt** [comment]
Opening the file from group folders works.

**Comment by b90g** [comment]
In addition, it works with group folders: AS LONG as they don't have an ampersand inside the name, `&` gets translated into `%26` inside our URL pattern which probably breaks the whole connector thing between nextcloud dav and keeweb thing.

see attached file:
![keewebampsign](https://user-images.githubusercontent.com/63392812/79254566-35684f00-7e85-11ea-827a-ae55b2e77f99.png)
the folder name was `KeeWeb&Testordner` which got translated into `KeeWeb%26Testordner` cut off after `KeeWeb` - Spaces between the words work fine btw.
